pkg/session_recording/recordings.jsx

```diff
@@ -425,13 +491,31 @@
         }
 
         render() {
-            if (this.state.recordingID === null) {
-                return <RecordingList list={this.state.recordingList} />;
-            } else {
+            const dateSince = this.state.dateSince;
+            const dateUntil = this.state.dateUntil;
+
+            if(this.state.recordingID === null) {
                 return (
-                    <Recording
-                        recording={this.recordingMap[this.state.recordingID]}
-                    />
+                    <div>
+                        <div className="content-header-extra">
+                            <table class="form-table-ct">
+                                <tr>
+                                    <td className="top"><label className="control-label" for="dateSince">Date Since</label></td>
+                                    <td><Datepicker element={'dateSince'} date={dateSince} onDateChange={this.handleDateSinceChange} /></td>
+                                    <td className="top"><label className="control-label" for="dateUntil">Date Until</label></td>
+                                    <td><Datepicker element={'dateUntil'} date={dateUntil} onDateChange={this.handleDateUntilChange} /></td>
+                                </tr>
+                            </table>
+                        </div>
```

Comment:
I think the markup above should be a part of the RecordingList component, and the hooks should be passed through it to Datepickers. This way the details of the markup could be hidden from View and the RecordingList could better integrate with it.